src/darwin/Framework/CHIP/MTRDeviceControllerXPCConnection.mm

```diff
@@ -195,7 +195,7 @@ - (void)handleReportWithController:(id)controller
         if (!controllerDictionary) {
             return;
         }
-        NSNumber * nodeIdKey = [NSNumber numberWithUnsignedInteger:nodeId];
+        NSNumber * nodeIdKey = [NSNumber numberWithUnsignedLongLong:nodeId];
```

Comment:
Is there any reason to not just do:
```suggestion
        NSNumber * nodeIdKey = @(nodeId);
```
and not mess with the `numberWithGuessWhatItShouldBe` bits?

Reply:
Done.